measure

```diff
@@ -17,12 +17,13 @@ HAS_CANCEL=True
 PROGRESS_INTERVAL=30
 
 DFLT_LOAD_CFG = {
-    'n_threads'  : 10,
-    'n_requests' : 100000000,   # request limit
-    't_limit'    : 180,         # time limit in seconds:   0 => no time limit
-    't_warmup'   :  30,         # warmup time in seconds:  0 => no warmup
-    'user'       : '',
-    'password'   : '',
+    'n_threads'     : 10,
+    'n_requests'    : 10000000,   # request limit
+    't_limit'       : 180,         # time limit in seconds:   0 => no time limit
+    't_warmup'      :  30,         # warmup time in seconds:  0 => no warmup
+    'user'          : '',
+    'password'      : '',
+    'extra_headers' : [],
```

Comment:
why do we call them "extra" headers; I would propose calling it simply "headers" everywhere (env var and load parameter)